addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3094,13 +3094,26 @@ export class OdooEditor extends EventTarget {
             styleSection.querySelector('button span').textContent = activeLabel;
         }
 
+        const isInMedia = this.toolbar.classList.contains('oe-media');
         const linkNode = getInSelection(this.document, 'a');
         const linkButton = this.toolbar.querySelector('#create-link');
         linkButton && linkButton.classList.toggle('active', !!linkNode);
+        // hide unlink button if no link in selection, always hide on media elements
+        const unlinkButton = this.toolbar.querySelector('#unlink');
+        unlinkButton && unlinkButton.classList.toggle('d-none', isInMedia || !linkNode);
         const undoButton = this.toolbar.querySelector('#undo');
         undoButton && undoButton.classList.toggle('disabled', !this.historyCanUndo());
         const redoButton = this.toolbar.querySelector('#redo');
         redoButton && redoButton.classList.toggle('disabled', !this.historyCanRedo());
+
+        // hide create-link button if selection spans several blocks, always hide on media elements
+        const range = getDeepRange(this.editable, { sel, correctTripleClick: true });
```

Comment:
Why do you need to use `getDeepRange` here?

Reply:
Good question!, Let's consider this scenario, `<p>[test]</p><p><123</test>`. If user did selection through triple click, then `spansBlocks` will be true but selection is still there at the first paragraph. So I though correcting triple click will be better here. What's your opinion here? Should we remain it as it was before?

Reply:
Ok! I think you're right! Let's keep it like this.